src/mapfile2js/parse/resolveSymbolset.ts

```diff
@@ -53,6 +53,27 @@ export function resolveSymbolsFromMapfile(mapfile: Mapfile): Mapfile {
                 });
               }
             });
+          }  else if (mclass.labels) {
+            // parse symbol data within a style tag of a label
+            mclass.labels.forEach((label) => {
+              const styles: MapfileStyle[] = label.styles as MapfileStyle[];
+              if (styles) {
+                styles.forEach((style: MapfileStyle) => {
+                  if (style.symbol) {
+                    symbols.forEach((symbol: any) => {
```

Comment:
You might use optional chaining here (`?.`)

Reply:
Changed into an optional chaining now.